src/test/java/net/datafaker/providers/base/BrandTest.java

```diff
@@ -0,0 +1,30 @@
+package net.datafaker.providers.base;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class BrandTest extends BaseFakerTest<BaseFaker> {
+
+    @Test
+    void testSport() {
+        assertThat(faker.brand().sport()).isNotEmpty();
```

Comment:
It's better to follow the new way for tests like e.g. done within this method https://github.com/datafaker-net/datafaker/blob/34c43b81889077d679589be8adef8102d78d9acb/src/test/java/net/datafaker/providers/base/AviationTest.java#L32-L37